examples/scaling-criteo/03-Training-with-Merlin-Models.ipynb

```diff
@@ -0,0 +1,417 @@
+{
```

Comment:
<p>better to change this to <code>model.save(os.path.join(input_path, "dlrm"))</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/380/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>